transport/http/inbound.go

```diff
@@ -171,6 +174,12 @@ func (i *Inbound) start() error {
 	}
 
 	i.addr = i.server.Listener().Addr().String() // in case it changed
+	if i.logger != nil {
+		i.logger.Info("http inbound started", zap.String("address", i.addr))
+		if len(i.router.Procedures()) == 0 {
+			i.logger.Warn("no procedures specified for http inbound")
+		}
+	}
```

Comment:
Rather than putting nil checks all over, can we use `zap.NewNop` by default?

Reply:
Done